% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
]{article}
\usepackage{amsmath,amssymb}
\usepackage{lmodern}
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\usepackage[margin=1.0in]{geometry}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{-\maxdimen} % remove section numbering
\newlength{\cslhangindent}
\setlength{\cslhangindent}{1.5em}
\newlength{\csllabelwidth}
\setlength{\csllabelwidth}{3em}
\newlength{\cslentryspacingunit} % times entry-spacing
\setlength{\cslentryspacingunit}{\parskip}
\newenvironment{CSLReferences}[2] % #1 hanging-ident, #2 entry spacing
 {% don't indent paragraphs
  \setlength{\parindent}{0pt}
  % turn on hanging indent if param 1 is 1
  \ifodd #1
  \let\oldpar\par
  \def\par{\hangindent=\cslhangindent\oldpar}
  \fi
  % set entry spacing
  \setlength{\parskip}{#2\cslentryspacingunit}
 }%
 {}
\usepackage{calc}
\newcommand{\CSLBlock}[1]{#1\hfill\break}
\newcommand{\CSLLeftMargin}[1]{\parbox[t]{\csllabelwidth}{#1}}
\newcommand{\CSLRightInline}[1]{\parbox[t]{\linewidth - \csllabelwidth}{#1}\break}
\newcommand{\CSLIndent}[1]{\hspace{\cslhangindent}#1}
\usepackage{booktabs}
\usepackage{longtable}
\usepackage{array}
\usepackage{multirow}
\usepackage{wrapfig}
\usepackage{float}
\usepackage{colortbl}
\usepackage{pdflscape}
\usepackage{tabu}
\usepackage{threeparttable}
\usepackage{threeparttablex}
\usepackage[normalem]{ulem}
\usepackage{makecell}
\usepackage{setspace}
\doublespacing
\usepackage[left]{lineno}
\usepackage[none]{hyphenat}
\raggedright
\usepackage{helvet} % Helvetica font
\renewcommand*\familydefault{\sfdefault} % Use the sans serif version of the font
\usepackage[T1]{fontenc}
\ifLuaTeX
  \usepackage{selnolig}  % disable illegal ligatures
\fi
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\urlstyle{same} % disable monospaced font for URLs
\hypersetup{
  hidelinks,
  pdfcreator={LaTeX via pandoc}}

\author{}
\date{\vspace{-2.5em}}

\begin{document}

\vspace*{10mm}

\hypertarget{the-riffomonas-youtube-channel-an-educational-resource-to-foster-reproducible-research-practices}{%
\section{\texorpdfstring{The \emph{Riffomonas} YouTube Channel: An
Educational Resource to Foster Reproducible Research
Practices}{The Riffomonas YouTube Channel: An Educational Resource to Foster Reproducible Research Practices}}\label{the-riffomonas-youtube-channel-an-educational-resource-to-foster-reproducible-research-practices}}

\vspace{15mm}

Running title: \emph{Riffomonas} YouTube Channel

\vspace{15mm}

Patrick D. Schloss\({^1}\)\({^\dagger}\)

\vspace{40mm}

\(\dagger\) To whom correspondence should be addressed:
\href{mailto:pschloss@umich.edu}{pschloss@umich.edu}

\(1\) Department of Microbiology and Immunology, University of Michigan,
Ann Arbor, MI 48109

\vspace{35mm}

\hypertarget{educational-resource}{%
\subsubsection{Educational resource}\label{educational-resource}}

\newpage
\linenumbers

\hypertarget{abstract}{%
\subsection{Abstract}\label{abstract}}

Methods for analyzing data in a reproducible manner are often viewed as
impenetrable to scientists more familiar with laboratory research. The
\emph{Riffomonas} YouTube channel is committed to teaching these
scientists and others how to engage in reproducible research using
modern data science tools.

\newpage

As high throughput data generation becomes more common in microbiology
and other disciplines there is a significant need for laboratory
scientists to develop data science skills (1). Unfortunately,
traditional undergraduate and graduate biology training programs are
often deficient in opportunities for scientists to develop the skills
necessary to analyze large datasets in a reproducible and robust manner
(2, 3). Numerous organizations seek to fill this void including the
Carpentries, Codeacademy, and DataCamp (4). There are also numerous
video tutorials available on YouTube. Although the content available
through these platforms are popular, there has been a gap in content
that emphasizes project-based learning.

The \emph{Riffomonas} YouTube channel
(\url{https://www.youtube.com/c/RiffomonasProject}) seeks to fill this
gap. I started consistently posting videos at the beginning of the
COVID-19 pandemic in the Spring of 2020. As of the end of November 2022,
the channel included 285 videos that had been viewed 635,947 times; the
channel had 11,327 subscribers. The majority of these are 264 videos in
the ``Code Club'' playlist (5). Other videos are related to a previously
described tutorial series on reproducible research (6) and series where
reproducible reseach practices are used to address topical questions.
Code Club videos are typically between 20 and 30 minutes long. The code
that is developed in the videos is available through a website
(\url{https://riffomonas.org/code_club/}) and the channel's
GitHub-hosted account (\url{https://github.com/riffomonas}).

The name, \emph{Riffomonas}, comes from the concept of ``riffing'' where
musical themes are adapted to achieve a similar sound, albeit perhaps in
a different context (6). This is to emphasize the value of
reproducibility not only to recreate a set of results but to apply a
method with a different dataset (7). The channel covers topics related
to reproducible data analysis practices including R programming, data
visualization, project organization, version control, command line
programming, workflow tools, and scientific publishing. Each video
includes a brief introduction followed by me live coding to achieve a
goal. I emphasize the use of live coding to modulate the rate of
instruction and to show viewers my own coding practices. Observing a
experienced analyst make mistakes normalizes some level of failure and
demonstrates the strategies they can use to resolve their own mistakes.
Viewers are encouraged to follow along with each video and to apply the
new information to their own project.

Each video emphasizes a specific topic, but includes other content that
is selected to review topics covered in recent videos. Although videos
can be watched individually, they often form a project arc. For example,
between July 2020 and July 2021, I formulated a research question,
obtained and analyzed data to answer the question, and wrote a paper
that was published in \emph{mSphere} (8). This series of 67 videos
covered every topic from creating the initial directory on my computer
to house the project files through reviewing the proofs of the published
manuscript. Other project arcs have included visualizing microbiome
data, modeling microbiome data using machine learning tools, analyzing
the impacts of rarefying microbiome data, and other topics. Going
forward, the \emph{Riffomonas} channel will continue to post
project-based content to help researchers develop their reproducible
research skills.

\hypertarget{acknowledgements}{%
\subsection{Acknowledgements}\label{acknowledgements}}

I am grateful to the audience of the \emph{Riffomonas} channel for their
feedback on topics that I should cover in future episodes.

\newpage

\hypertarget{references}{%
\subsection{References}\label{references}}

\hypertarget{refs}{}
\begin{CSLReferences}{0}{1}
\leavevmode\vadjust pre{\hypertarget{ref-Barone2017}{}}%
\CSLLeftMargin{1. }%
\CSLRightInline{\textbf{Barone L}, \textbf{Williams J}, \textbf{Micklos
D}. 2017. Unmet needs for analyzing biological big data: A survey of 704
{NSF} principal investigators. {PLOS} Computational Biology
\textbf{13}:e1005755.
doi:\href{https://doi.org/10.1371/journal.pcbi.1005755}{10.1371/journal.pcbi.1005755}.}

\leavevmode\vadjust pre{\hypertarget{ref-Schloss2018b}{}}%
\CSLLeftMargin{2. }%
\CSLRightInline{\textbf{Schloss PD}. 2018. Identifying and overcoming
threats to reproducibility, replicability, robustness, and
generalizability in microbiome research. {mBio} \textbf{9}.
doi:\href{https://doi.org/10.1128/mbio.00525-18}{10.1128/mbio.00525-18}.}

\leavevmode\vadjust pre{\hypertarget{ref-Williams2019}{}}%
\CSLLeftMargin{3. }%
\CSLRightInline{\textbf{Williams JJ}, \textbf{Drew JC},
\textbf{Galindo-Gonzalez S}, \textbf{Robic S}, \textbf{Dinsdale E},
\textbf{Morgan WR}, \textbf{Triplett EW}, \textbf{Burnette JM},
\textbf{Donovan SS}, \textbf{Fowlks ER}, \textbf{Goodman AL},
\textbf{Grandgenett NF}, \textbf{Goller CC}, \textbf{Hauser C},
\textbf{Jungck JR}, \textbf{Newman JD}, \textbf{Pearson WR},
\textbf{Ryder EF}, \textbf{Sierk M}, \textbf{Smith TM},
\textbf{Tosado-Acevedo R}, \textbf{Tapprich W}, \textbf{Tobin TC},
\textbf{Toro-Martínez A}, \textbf{Welch LR}, \textbf{Wilson MA},
\textbf{Ebenbach D}, \textbf{McWilliams M}, \textbf{Rosenwald AG},
\textbf{Pauley MA}. 2019. Barriers to integration of bioinformatics into
undergraduate life sciences education: A national study of {US} life
sciences faculty uncover significant barriers to integrating
bioinformatics into undergraduate instruction. {PLOS} {ONE}
\textbf{14}:e0224288.
doi:\href{https://doi.org/10.1371/journal.pone.0224288}{10.1371/journal.pone.0224288}.}

\leavevmode\vadjust pre{\hypertarget{ref-Wilson2016}{}}%
\CSLLeftMargin{4. }%
\CSLRightInline{\textbf{Wilson G}. 2016. Software carpentry: Lessons
learned. F1000Research.
doi:\href{https://doi.org/10.12688/f1000research.3-62.v2}{10.12688/f1000research.3-62.v2}.}

\leavevmode\vadjust pre{\hypertarget{ref-Hagan2020}{}}%
\CSLLeftMargin{5. }%
\CSLRightInline{\textbf{Hagan AK}, \textbf{Lesniak NA}, \textbf{Balunas
MJ}, \textbf{Bishop L}, \textbf{Close WL}, \textbf{Doherty MD},
\textbf{Elmore AG}, \textbf{Flynn KJ}, \textbf{Hannigan GD},
\textbf{Koumpouras CC}, \textbf{Jenior ML}, \textbf{Kozik AJ},
\textbf{McBride K}, \textbf{Rifkin SB}, \textbf{Stough JMA},
\textbf{Sovacool KL}, \textbf{Sze MA}, \textbf{Tomkovich S},
\textbf{Topcuoglu BD}, \textbf{Schloss PD}. 2020. Ten simple rules to
increase computational skills among biologists with code clubs. {PLOS}
Computational Biology \textbf{16}:e1008119.
doi:\href{https://doi.org/10.1371/journal.pcbi.1008119}{10.1371/journal.pcbi.1008119}.}

\leavevmode\vadjust pre{\hypertarget{ref-Schloss2018a}{}}%
\CSLLeftMargin{6. }%
\CSLRightInline{\textbf{Schloss PD}. 2018. The {R}iffomonas reproducible
research tutorial series. Journal of Open Source Education
\textbf{1}:13.
doi:\href{https://doi.org/10.21105/jose.00013}{10.21105/jose.00013}.}

\leavevmode\vadjust pre{\hypertarget{ref-Leek2015}{}}%
\CSLLeftMargin{7. }%
\CSLRightInline{\textbf{Leek JT}, \textbf{Peng RD}. 2015. Reproducible
research can still be wrong: Adopting a prevention approach. Proceedings
of the National Academy of Sciences \textbf{112}:1645--1646.
doi:\href{https://doi.org/10.1073/pnas.1421412111}{10.1073/pnas.1421412111}.}

\leavevmode\vadjust pre{\hypertarget{ref-Schloss2021}{}}%
\CSLLeftMargin{8. }%
\CSLRightInline{\textbf{Schloss PD}. 2021. Amplicon sequence variants
artificially split bacterial genomes into separate clusters. {mSphere}
\textbf{6}.
doi:\href{https://doi.org/10.1128/msphere.00191-21}{10.1128/msphere.00191-21}.}

\end{CSLReferences}

\end{document}
